cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -906,6 +948,7 @@ pub(crate) fn define(
         refer to a byte inside the stack slot:
         `0 <= Offset < sizeof(SS)`.
         "#,
+            &formats.stack_load,
```

Comment:
I think the stack_load format should be renamed, as it is used for stack_addr here too.